src/connectors/refinement-list/connectRefinementList.js

```diff
@@ -374,11 +388,9 @@ export default function connectRefinementList(renderFn, unmountFn = noop) {
         unmountFn();
 
         if (operator === 'and') {
-          return state.removeFacetRefinement(attribute).removeFacet(attribute);
+          return state.removeFacet(attribute);
         } else {
```

Comment:
The `else` branch is useless since we early return inside the `if`.

Reply:
92904a8de